bin/check_tron_jobs.py

```diff
@@ -6,17 +6,27 @@
 import sys
 import time
 
+from enum import Enum
 from pysensu_yelp import send_event
 
 from tron.commands import cmd_utils
 from tron.commands import display
 from tron.commands.client import Client
 from tron.commands.client import get_object_type_from_identifier
 
-
 log = logging.getLogger('check_tron_jobs')
 
 
+class State(Enum):
+    SUCCESSED = "succeeded"
```

Comment:
just do succeeded. `SUCCESSED` is not a word (but kinda looks like it should be)